From: Shengjing Zhu <zhsj@debian.org>
Date: Thu, 8 Oct 2020 02:39:10 +0800
Subject: only warn on unsupported arch

Forwarded: not-needed
---
 etcdmain/etcd.go | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/etcdmain/etcd.go b/etcdmain/etcd.go
index 73328a7..f2942d4 100644
--- a/etcdmain/etcd.go
+++ b/etcdmain/etcd.go
@@ -614,6 +614,5 @@ func checkSupportArch() {
 		return
 	}
 
-	fmt.Printf("etcd on unsupported platform without ETCD_UNSUPPORTED_ARCH=%s set\n", runtime.GOARCH)
-	os.Exit(1)
+	plog.Warningf("etcd on unsupported platform without ETCD_UNSUPPORTED_ARCH=%s set\n", runtime.GOARCH)
 }
